src/clj/kamera/devcards.clj

```diff
@@ -100,6 +100,7 @@
      (let [target-urls (find-test-urls session)
            targets (map (fn [target-url]
                           {:url (str devcards-url target-url)
+                           {:resize-to-contents {:dom-selector "#com-rigsomelight-devcards-main"}}
```

Comment:
This looks like it's got an extra pair of braces? I also just realised circle wasn't building PRs, I've enabled that now

Reply:
Where do you see an extra pair of braces?

Reply:
`:resize-to-contents` should be a sibling of `:url`, i.e. the options map should be 
```
:url "..."
:resize-to-contents {:dom-selector "..."}
```

Or am I just reading totally wrong? (I'm on my phone)

Reply:
Oh, I’m sorry. You are referring to the outer braces. Yes, those are wrong.